nix/tests.nix

```diff
@@ -81,9 +81,9 @@ let
         rootdir="$(${git}/bin/git rev-parse --show-toplevel)"
         cd "$rootdir"
 
-        export PATH="${postgrestStatic}/bin:${curl}/bin:${ncat}/bin:$PATH"
```

Comment:
The ncat/nmap dependency was removed in #1630 already, so we don't need it anymore.